libpromises/verify_reports.c

```diff
@@ -165,7 +165,28 @@ static bool PrintFile(const char *filename, size_t max_lines)
     size_t line_size = CF_BUFSIZE;
     char *line = xmalloc(line_size);
 
-    for (size_t i = 0; i < max_lines; i++)
+    int skip_lines = 0;
+    if (max_lines < 0)
+    {
+        skip_lines = max_lines;
+        max_lines = labs(max_lines);
```

Comment:
I think it would be better to use a macro here to avoid dependency on `ssize_t` type definition. `long int` might not be enough.

Reply:
So `max_lines` is `ssize_t` and `labs()` returns `long`, as Vratislav is saying, it's better to avoid the cast here, and use something that is `ssize_t` on the right side.